Auth0/_ObjectiveAuthenticationAPI.swift

```diff
@@ -47,6 +47,13 @@ public class _ObjectiveAuthenticationAPI: NSObject {
             .login(usernameOrEmail: username, password: password, connection: connection, scope: scope, parameters: parameters ?? [:])
             .start(handleResult(callback: callback))
     }
+    
+    @objc(loginWithUsernameOrEmail:password:realm:audience:scope:parameters:callback:)
+    public func login(withUsernameOrEmail username: String, password: String, realm: String, audience: String, scope: String, parameters: [String: Any]?, callback: @escaping (NSError?, Credentials?) -> Void) {
```

Comment:
This triggers a swiftlint warning. Please add `// swiftlint:disable:next function_parameter_count`.